gensim/models/ldaseqmodel.py

```diff
@@ -20,6 +20,19 @@
         - in particular, update_obs and the optimization takes a lot time.
     4) Try and make it distributed, especially around the E and M step.
     5) Remove all C/C++ coding style/syntax.
+
+Examples
+--------
+
+#. Set up a model using have 30 documents, with 5 in the first time-slice, 10 in the second, and 15 in the third:
+
+>>> from gensim.test.utils import common_corpus
+>>> ldaseq = LdaSeqModel(corpus=common_corpus, time_slice=[2, 4, 3], num_topics=2)
+
+#. Persist model to disk:
+
+>>> ldaseq.save("ldaseq")
```

Comment:
Please add also "how to get an vectors" to example, this is important always (for all examples)

Reply:
You mean how to get the vector representation of a single document right? I will add an example.